packages/mobile/src/components/app-navigator/BottomTabNavigator.tsx

```diff
@@ -93,6 +94,10 @@ const ProfileStackScreen = createStackScreen<ProfileStackParamList>(Stack => (
     <Stack.Screen name='EditProfile' component={EditProfileScreen} />
     <Stack.Screen name='SettingsScreen' component={SettingsScreen} />
     <Stack.Screen name='AboutScreen' component={AboutScreen} />
+    <Stack.Screen
+      name='ListeningHistoryScreen'
```

Comment:
ah these screens have incorrect names above, standard is to not put "Screen" in the name, can maybe address later